testnet/stacks-node/src/run_loop/neon.rs

```diff
@@ -542,33 +550,47 @@ impl RunLoop {
         let (coordinator_thread_handle, attachments_rx) =
             self.spawn_chains_coordinator(&burnchain_config, coordinator_receivers);
         self.instantiate_pox_watchdog();
+        self.start_prometheus();
 
         // We announce a new burn block so that the chains coordinator
         // can resume prior work and handle eventual unprocessed sortitions
         // stored during a previous session.
         coordinator_senders.announce_new_burn_block();
 
-        // Wait for some sortitions!
-        let mut burnchain_tip = burnchain
-            .wait_for_sortitions(None)
-            .expect("Unable to get burnchain tip");
+        // Make sure at least one sortition has happened
+        let sortdb = burnchain.sortdb_mut();
+        let (rc_aligned_height, sn) =
+            RunLoop::get_reward_cycle_sortition_db_height(&sortdb, &burnchain_config);
+
+        let burnchain_tip_snapshot = if sn.block_height == burnchain_config.first_block_height {
+            // need at least one sortition to happen.
+            burnchain
+                .wait_for_sortitions(Some(1))
```

Comment:
I think `wait_for_sortitions` uses real block height, not "sortition height" for its argument.

Reply:
Yup; fixed